pyproject.toml

```diff
@@ -34,6 +34,7 @@ watchdog = ">= 3.0.0, < 4"
 pandas = {version = ">= 2.2.0, < 3", optional = true}
 pyarrow = {version = ">= 15.0.0, < 16.0.0",optional = true}
 plotly = {version = ">= 5.18.0, < 6", optional = true}
+scikit-learn = {version = "^1.4.1.post1", optional = true}
```

Comment:
Why is this here? Do we require it now for any reason?

Reply:
We are using it in the getting started application.